### PR TITLE
German string for viewrestrictions

### PR DESCRIPTION
**Pull request by nic547**
**When merged this pull request will:**
Add the german entries to the stringtable.xml for the viewrestriction

### CODE REVIEW

File: addons/viewrestriction/stringtable.xml [hunk 1]
```diff
@@ -4,74 +4,92 @@
...
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_Mode">
             <English>Mode</English>
             <Polish>Tryb</Polish>
+            <German>Modus</German>
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeDesc">
             <English>Sets global mode. Default: Disabled</English>
             <Polish>Zmienia tryb globalny. (Domyślnie: Wyłączony)</Polish>
+            <German>Globaler Modus. (Standart: Deaktiviert)</German>
```
Comment (bux578) [inline addons/viewrestriction/stringtable.xml]: Standard. (Es geht hier nicht um die Art zu stehen) :P

File: addons/viewrestriction/stringtable.xml [hunk 2]
```diff
@@ -4,74 +4,92 @@
...
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveFoot">
             <English>(Selective) Foot</English>
             <Polish>(Tryb selektywny) Piechota</Polish>
+            <German>(Selektiv) zu Fuß</German>
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveFootDesc">
             <English>Selective mode on Foot. Default: Disabled (Requires Mode: Selective)</English>
             <Polish>Tryb selektywny piechoty. Domyślnie: Wyłączony (Wymagany tryb globalny: Selektywny)</Polish>
+            <German>Selektiver Modus zu Fuß. (Standart: Deaktiviert, Benötigt Modus: Selektiv)</German>
```
Comment (bux578) [inline addons/viewrestriction/stringtable.xml]: Standard

File: addons/viewrestriction/stringtable.xml [hunk 3]
```diff
@@ -4,74 +4,92 @@
...
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveLand">
             <English>(Selective) Land Vehicles</English>
             <Polish>(Tryb selektywny) Pojazdy</Polish>
+            <German>(Selektiv) Landfahrzeuge</German>
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveLandDesc">
             <English>Selective mode in Land Vehicles. Default: Disabled (Requires Mode: Selective)</English>
             <Polish>Tryb selektywny pojazdów. Domyślnie: Wyłączony (Wymagany tryb globalny: Selektywny)</Polish>
+            <German>Selektiver Modus in Landfahrzeugen. (Standart: Deaktiviert, Benötigt Modus: Selektiv)</German>
```
Comment (bux578) [inline addons/viewrestriction/stringtable.xml]: Standard

File: addons/viewrestriction/stringtable.xml [hunk 4]
```diff
@@ -4,74 +4,92 @@
...
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveAir">
             <English>(Selective) Air Vehicles</English>
             <Polish> (Tryb selektywny) Lotnictwo</Polish>
+            <German>(Selektiv) Luftfahrzeuge</German>
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveAirDesc">
             <English>Selective mode in Air Vehicles. Default: Disabled (Requires Mode: Selective)</English>
             <Polish>Tryb selektywny lotnictwa. Domyślnie: Wyłączony (Wymagany tryb globalny: Selektywny)</Polish>
+            <German><German>Selektiver Modus in Luftfahrzeugen. (Standart: Deaktiviert, Benötigt Modus: Selektiv)</German></German>
```
Comment (bux578) [inline addons/viewrestriction/stringtable.xml]: Standard

File: addons/viewrestriction/stringtable.xml [hunk 5]
```diff
@@ -4,74 +4,92 @@
...
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveSea">
             <English>(Selective) Sea Vehicles</English>
             <Polish>(Tryb selektywny) Jednostki pływające</Polish>
+            <German>(Selektiv) Wasserfahrzeuge</German>
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveSeaDesc">
             <English>Selective mode in Sea Vehicles. Default: Disabled (Requires Mode: Selective)</English>
             <Polish>Tryb selektywny jednostek pływających. Domyślnie: Wyłączony (Wymagany tryb globalny: Selektywny)</Polish>
+            <German><German>Selektiver Modus zu Fuß. (Standart: Deaktiviert, Benötigt Modus: Selektiv)</German></German>
```
Comment (bux578) [inline addons/viewrestriction/stringtable.xml]: Standard
`<German><German></German></German>`

File: addons/viewrestriction/stringtable.xml [hunk 6]
```diff
@@ -4,74 +4,92 @@
...
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveUAV">
             <English>(Selective) UAVs</English>
             <Polish> (Tryb selektywny) UAV</Polish>
+            <German>(Selektiv) Unbemannte Luftfahrzeuge</German>
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveUAVDesc">
             <English>Selective mode in UAVs. Default: Disabled (Requires Mode: Selective)</English>
             <Polish>Tryb selektywny UAV. Domyślnie: Wyłączony (Wymagany tryb globalny: Selektywny)</Polish>
+            <German>Selektiver Modus in unbemannten Luftfahrzeugen. (Standart: Deaktiviert, Benötigt Modus: Selektiv</German>
```
Comment (bux578) [inline addons/viewrestriction/stringtable.xml]: Standard

**Comment by nic547** [comment]
I'm a hopeless case ^^

File: addons/viewrestriction/stringtable.xml [hunk 7]
```diff
@@ -4,74 +4,92 @@
...
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveSea">
             <English>(Selective) Sea Vehicles</English>
             <Polish>(Tryb selektywny) Jednostki pływające</Polish>
+            <German>(Selektiv) Wasserfahrzeuge</German>
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveSeaDesc">
             <English>Selective mode in Sea Vehicles. Default: Disabled (Requires Mode: Selective)</English>
             <Polish>Tryb selektywny jednostek pływających. Domyślnie: Wyłączony (Wymagany tryb globalny: Selektywny)</Polish>
+            <German><German>Selektiver Modus zu Fuß. (Standard: Deaktiviert, Benötigt Modus: Selektiv)</German></German>
```
Comment (jonpas) [inline addons/viewrestriction/stringtable.xml]: Still double tags.

File: addons/viewrestriction/stringtable.xml [hunk 8]
```diff
@@ -4,74 +4,92 @@
...
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveAir">
             <English>(Selective) Air Vehicles</English>
             <Polish> (Tryb selektywny) Lotnictwo</Polish>
+            <German>(Selektiv) Luftfahrzeuge</German>
         </Key>
         <Key ID="STR_ACEX_ViewRestriction_ModeSelectiveAirDesc">
             <English>Selective mode in Air Vehicles. Default: Disabled (Requires Mode: Selective)</English>
             <Polish>Tryb selektywny lotnictwa. Domyślnie: Wyłączony (Wymagany tryb globalny: Selektywny)</Polish>
+            <German><German>Selektiver Modus in Luftfahrzeugen. (Standard: Deaktiviert, Benötigt Modus: Selektiv)</German></German>
```
Comment (jonpas) [inline addons/viewrestriction/stringtable.xml]: Still double tags.

**Pull request merged**